### DISCUSSION TITLE
errore nella procedura di configurazione automatica https

### DISCUSSION THREAD

**Discussion by matcor73** [globaleaks/globaleaks-whistleblowing-software#3541] [Q&A]
### What version of GlobaLeaks are you using?

4.11.5

### What browser(s) are you seeing the problem on?

Tor Browser

### What operating system(s) are you seeing the problem on?

Linux

### Describe the issue

come amministratore vado in Rete e poi sotto il tab https clicco su procedi sotto "configurazione automatica".
ottengo un errore generico:

![image](https://github.com/globaleaks/GlobaLeaks/assets/140150827/7b0b1bc5-db65-4285-bad6-524a1df0eae1)

nel file /var/globaleaks/log/globaleaks.log:

2023-07-21 09:53:47+0200 [-] [E] exception mail suppressed for exception (<class 'acme.errors.ValidationError'>) [reason: special exception]

**Comment by evilaliv3** [comment]
@matcor73 : have you configured the dns pointing the domain name to the ip of the server?

**Comment by matcor73** [comment]
> @matcor73 : have you configured the dns pointing the domain name to the ip of the server?

@evilaliv3  yes, I did it

**Comment by evilaliv3** [comment]
Thank you for the reply.

If the DNS propagated the other possible situation to check is that your
server is exposed on internet on both port 80 and 443.

**Reply by matcor73** [reply]
Something strange here, almost from my point of view.

We have a proxmox vm configured with two ips, one public and another one for vpn access only.

Before I installed Globaleaks I was able to connect via ssh both way, public ip and private ip. Now I noticed I can access only with the provate one.

Furthermore the ports 80 and 443 are visible from private but not from public. So it looks to me the globaleaks installation closed the public ports, which make sense for security reasons, but maybe close too much ?

Btw I can access the admin console using the tor browser so the public IP is not completely "closed".

I hope this can help to understand better the root cause of the problem,

thank you

**Comment by matcor73** [comment]
Hi,
I would like to inform you the problem was solved.

I reinstalled from scratch on a debian12 (instead of 11), and after that the automatic generation of certificates worked fine.

Thank you
